src/main/webapp/PMmodule/GenericIntake/Edit.jsp

```diff
@@ -441,11 +440,9 @@
                                 <td>
                                     <label>HC Type</label>
                                     <br/>
-                                    <select name="client.hcType">
-                                        <c:forEach var="province" items="${GenericIntakeConstants.getProvinces()}">
-                                            <option value="${province.value}">${province.label}</option>
```

Comment:
These were causing errors?

Reply:
Not really. I used a command to replace the <HTML:HTML> tag, and don't know why this file was messed up. I will revert this one.